### PR TITLE
Add support for "is-horizontal" in Field

### PR DESCRIPTION
**Pull request by electrolinux**
Add a horizontal prop to the Field component (along with a new FieldBody component to handle multiple controls in the slot.)

Let me know if you need more for reviewing this pr.

Here is a screenshot of the Field documentation page related to Horizontal
buefy-horizontal

### CODE REVIEW

File: src/components/field/Field.vue [hunk 1]
```diff
@@ -1,17 +1,33 @@
 <template>
     <div class="field" :class="[fieldType, newPosition, {
         'is-expanded': expanded,
-        'is-grouped-multiline': groupMultiline
+        'is-grouped-multiline': groupMultiline,
+        'is-horizontal': horizontal
     }]">
-        <label class="label" :for="labelFor" v-if="label">{{ label }}</label>
-        <slot></slot>
+        <div class="field-label is-normal" v-if="horizontal">
```
Comment (electrolinux) [inline src/components/field/Field.vue]: The "is-normal" class add the required `padding-top` for aligning the label to the control text

File: src/components/field/FieldBody.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<script>
+    export default {
+        name: 'fieldBody',
+        render(h) {
+            return h('div', { attrs: { 'class': 'field-body' }}, this.$slots.default.map(v => {
+                // skip returns and comments
+                if (!v.tag) {
+                    return v
+                }
+                return h('div', { attrs: { 'class': 'field' }}, [v])
```
Comment (rafaelpimpa) [inline src/components/field/FieldBody.vue]: Lets use template here instead of render function to keep a pattern.

Comment (electrolinux) [reply src/components/field/FieldBody.vue]: I'll be happy to do that, if only I knew how...
The whole (and unique) purpose of this FieldBody component is to:
* enclose everything inside a `<div class="field-body">` tag
* wrap each `<slot/>` entries in a `<div class="field">`

Not doing the second point means we can't have more than one control in a horizontal field, while the [Bulma doc](https://bulma.io/documentation/form/general/#horizontal-form) show it is possible.

In fact, I've started without any FieldBody, using this template:

    <div class="field-body" v-if="horizontal">
        <div class="field"><slot></slot></div>
    </div>


But it doesn't works with more than one control in slot.

Comment (rafaelpimpa) [reply src/components/field/FieldBody.vue]: What's the difficult on doing that with template syntax? Not sure if I got the point.

Comment (electrolinux) [reply src/components/field/FieldBody.vue]: All I got with a template solution was:

    <div class="field-body">
        <div class="field">
            <input/>
            <input/>
            ...
        </div>
    </div

But what is really reeded is:

    <div class="field-body">
        <div class="field">
            <input/>
        </div>
        <div class="field">
            <input/>
        </div>
    </div>

And I really don't know how to do that with template only. 

Found this render solution on [stackoverflow](https://stackoverflow.com/questions/43834785/how-add-parent-for-each-slots-in-vuejs)

Comment (rafaelpimpa) [reply src/components/field/FieldBody.vue]: You are right, this is not easily done without render functions, my mistake.

File: src/components/field/FieldBody.vue [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<script>
+    export default {
+        name: 'fieldBody',
```
Comment (rafaelpimpa) [inline src/components/field/FieldBody.vue]: Update component name to ``bFieldBody``, and call like ``<b-field-body>`` :)

**Comment by electrolinux** [comment]
Requested changes applied :)

**Comment by rafaelpimpa** [comment]
Thanks!

**Pull request merged**